app/src/main/java/com/android/sample/ui/listActivities/LikedActivities.kt

```diff
@@ -80,17 +81,23 @@ fun LikedActivitiesScreen(
           when (uiState) {
             is ListActivitiesViewModel.ActivitiesUiState.Success -> {
               if (profile == null) {
-                Text(
-                    text = "You are not logged in, Login or Register to see your liked activities",
-                    modifier =
-                        Modifier.padding(8.dp)
-                            .align(Alignment.Center)
-                            .testTag("notConnectedPrompt"),
-                    color = MaterialTheme.colorScheme.onSurface)
-                Button(
-                    onClick = { navigationActions.navigateTo(Screen.SIGN_UP) },
-                    modifier = Modifier.testTag("signInButton")) {
-                      Text("Go to Sign In Page")
+                Column(
```

Comment:
Great to use a column. You allow for better responsiveness and a good looking UI.